### DISCUSSION TITLE
Where SMPLAYER saves the current stopping positing of each video?

### DISCUSSION THREAD

**Discussion by DannyFeldman** [smplayer-dev/smplayer#673] [Q&A]
Every time I open a video file, it starts just where I stopped last time. This is due to the checkbox "remember time position" in the preferences. 

Where can I find the list of current saved positions and their corresponding files?
smplayer.ini contains just few recents files and without their positon.

**Comment by smplayer-dev** [comment]
They are in the folder `file_settings`.

**Reply by DannyFeldman** [reply]
Thank you so much! Is there any chance to get the file names of my recent videos that appear there?

It seems that the filenames there is a hash that is based only on the video's content.
My goal is to get the filenames of my recent videos. While the "recent files" in the smplayer.ini contains only few names, the file_settings folder is huge.

**Comment by smplayer-dev** [comment]
If you look at the smplayer log while playing back a video you'll see the file that has been loaded:
`[10:00:17:954] FileSettingsHash::loadSettingsFor: config_file: "/home/...../file_settings/9/93065dbb9208e014.ini" `

**Reply by DannyFeldman** [reply]
Thanks, but I meant to the name of the video files.
The reason is that I lost many folders of my hard drive with all the videos, but smplayer still alive. I want to check which movies did I saw in the last year to buy them again.

**Comment by smplayer-dev** [comment]
Unfortunately those ini files don't store the filenames.
